http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -45,7 +66,8 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
                            samDao: SamDAO[F],
                            cromwellDao: CromwellDAO[F],
                            cbasDao: CbasDAO[F],
-                           wdsDao: WdsDAO[F]
+                           wdsDao: WdsDAO[F],
+                           apiClientCache: Cache[F, AKSClusterName, ApiClient]
```

Comment:
Do we really need a cache of k8s clients? I think the code in `AKSInterpreter.create/deleteApp` is already calling `getClusterCredential` every time to auth with helm. Could we just build a k8s client each time, and use it for the duration of the polling?

Reply:
Yeah I was thinking the same thing but this is how it is written in workbench libs -- I agree lets take it out